components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/internal/FrameworkServiceComponent.java

```diff
@@ -214,7 +214,13 @@ protected void activate(ComponentContext ctxt) {
         dataHolder.setJsFunctionRegistry(new JsFunctionRegistryImpl());
         BundleContext bundleContext = ctxt.getBundleContext();
 
-        bundleContext.registerService(JsFunctionRegistry.class, dataHolder.getJsFunctionRegistry(), null);
+        if (FrameworkUtils.isAdaptiveAuthenticationAvailable()) {
+            log.info("is Adaptive Authentication Enable : True");
```

Comment:
let's improve this log to more user friendly

Reply:
+1